src/index.js

```diff
@@ -662,6 +662,15 @@ const members = {
         return ` ${attr.name}${attr.value ? `="${attr.value}"` : ''}`;
       }).join('');
       return `<${name}${attributes}>${this.innerHTML}</${name}>`;
+    },
+
+    set (outerHTML) {
+      if (this.parentNode) {
+        const parsed = parse(outerHTML);
+        this.parentNode.replaceChild(parsed.firstChild, this);
+      } else {
+        throw new Error('Failed to set the \'outerHTML\' property on \'Element\': This element has no parent node');
```

Comment:
You're right, I didn't realise that.

```
var d = document.createElement('div'); d.outerHTML = '<em>i</em>'; console.log(d.outerHTML);

Uncaught DOMException: Failed to set the 'outerHTML' property on 'Element': This element has no parent node.
```

@nadiam84 , to make it closer, could we change it to `throw new DOMException`